frontend/src/assets/manual/en-GB/namedentities.md

```diff
@@ -0,0 +1,10 @@
+In some corpora, you have the option to view named entities. If this option is available, a toggle button labeled *Show named entities* is visible in the document overview, above the document content and metadata.
+
+Activating the toggle will show entities of four types:
+- Persons
+- Locations
+- Organizations
+- Miscellaneous
+
+### Disclaimer
```

Comment:
This should be a second-level header (a third-level header would be appropriate within a section that has a second-level header).

```suggestion
## Disclaimer
```